packages/embark/src/lib/core/engine.js

```diff
@@ -170,7 +170,7 @@ class Engine {
   }
 
   console(_options) {
-    this.registerModule('console', {
+    this.registerModulePackage('embark-console', {
```

Comment:
After splitting out the module into a package need to change `registerModule` -> `registerModulePackage`.